molgenis-core-ui/src/main/javascript/modules/react-components/AggregateTable.js

```diff
@@ -12,7 +14,7 @@
 	 * @memberOf component
 	 */
 	var AggregateTable = React.createClass({
-		mixins: [molgenis.ui.mixin.DeepPureRenderMixin],
+		mixins: [DeepPureRenderMixin],
```

Comment:
React folks are thinking about a solution for this. See https://facebook.github.io/react/docs/reusable-components.html

> Instead, we're working on making it easier to support such use cases without resorting to mixins.